### PR TITLE
Merge with main 

### PR DESCRIPTION
**Pull request by hugobcerdaminaya**
Merge with main, dark mode and desktop and tablet responsiveness

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<html lang="en" class="dark">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>nounamity</title>
+    <link rel="stylesheet" href="assets/css/styles.css">
+    <script src="https://cdn.tailwindcss.com"></script>
```
Comment (jvtrudel) [inline index.html:9]: do we need both styles.css and cdn.tailwindcss

**Review by jvtrudel — CHANGES_REQUESTED** [review]

File: assets/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,843 @@
+@import url('https://fonts.googleapis.com/css2?family=Arimo:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&display=swap');
+
+/*
+! tailwindcss v3.0.23 | MIT License | https://tailwindcss.com
```
[the remark this answers is not in the document]
Comment (hugobcerdaminaya) [reply assets/css/styles.css:4]: the command that generates it is 
npx tailwindcss -i ./src/styles.css -o ./public/assets/css/styles.css --watch

in the src/styles.css file you have the following imports, to generate the styles.css file in the public folder

Captura srcstyles

Captura tailwind_config

File: index.html (hunk 1 above)
Comment (hugobcerdaminaya) [reply index.html:9]: The line where the cdn is called will be removed, only the call to the styles.css file will remain, the script line for bg-color-about is also removed.

These changes are in a commit that I will upload shortly.

Captura index

File: index.html [hunk 3]
```diff
@@ -27,7 +27,7 @@
             <div class="container mx-auto grid grid-cols-5 gap-4">    
             <!--<div class="flex flex-grow">-->
                 <div class="col-span-3 pl-20">
-                    <img class="w-96" src="./assets/img/logo noumanity-white.png" alt="">
```
[the remark this answers is not in the document]
Comment (hugobcerdaminaya) [reply index.html]: @samuelsads  I'm reviewing and I believe those files were generated on your computer, I saw that when I checked the ._index file

Captura  _index

**Pull request merged**